test/bats/alibabacloud.bats

```diff
@@ -0,0 +1,151 @@
+#!/usr/bin/env bats
+
+load helpers
+
+WAIT_TIME=120
+SLEEP_TIME=1
+NAMESPACE=kube-system
+POD_NAME=alibabacloud-basic-test-mount
+BATS_TEST_DIR=test/bats/tests/alibabacloud
+
+setup() {
+  if [[ -z "${ALIBABACLOUD_ACCESS_KEY}" ]] || [[ -z "${ALIBABACLOUD_ACCESS_SECRET}" ]]; then
+    echo "Error: ram ak/sk is not provided" >&2
+    return 1
+  fi
+}
+
+setup_file() {
+    #Configure aliyun cli profile
+    aliyun configure set --profile akProfile --mode AK --region us-west-1 --access-key-id ${ALIBABACLOUD_ACCESS_KEY} --access-key-secret ${ALIBABACLOUD_ACCESS_SECRET}
+
+    #Create test secrets
+    aliyun kms CreateSecret --SecretName testBasic --SecretData testValue --VersionId v1
+    aliyun kms CreateSecret --SecretName testSync --SecretData  syncValue --VersionId v1
+    aliyun kms CreateSecret --SecretName testRotation --SecretData beforeRotation --VersionId v1
+    aliyun kms CreateSecret --SecretName testJson --SecretData '{"username": "testUser", "password": "testPassword"}' --VersionId v1
+}
+
+teardown_file() {
+    aliyun kms DeleteSecret --SecretName testBasic --ForceDeleteWithoutRecovery true
+    aliyun kms DeleteSecret --SecretName testSync --ForceDeleteWithoutRecovery true
+    aliyun kms DeleteSecret --SecretName testRotation --ForceDeleteWithoutRecovery true
+    aliyun kms DeleteSecret --SecretName testJson --ForceDeleteWithoutRecovery true
+}
+
+validate_jsme_mount() {
+    result=$(kubectl --namespace $NAMESPACE exec $POD_NAME -- cat /mnt/secrets-store/$USERNAME_ALIAS)
+    [[ "${result//$'\r'}" == $USERNAME ]]
+
+    result=$(kubectl --namespace $NAMESPACE exec $POD_NAME -- cat /mnt/secrets-store/$PASSWORD_ALIAS)
+    [[ "${result//$'\r'}" == $PASSWORD ]]
+
+    result=$(kubectl --namespace $NAMESPACE exec $POD_NAME -- cat /mnt/secrets-store/$SECRET_FILE_NAME)
+    [[ "${result//$'\r'}" == $SECRET_FILE_CONTENT ]]
+
+    run kubectl get secret --namespace $NAMESPACE $K8_SECRET_NAME
+    [ "$status" -eq 0 ]
+
+    result=$(kubectl --namespace=$NAMESPACE get secret $K8_SECRET_NAME -o jsonpath="{.data.username}" | base64 -d)
+    [[ "$result" == $USERNAME ]]
+
+    result=$(kubectl --namespace=$NAMESPACE get secret $K8_SECRET_NAME -o jsonpath="{.data.password}" | base64 -d)
+    [[ "$result" == $PASSWORD ]]
+}
+
+@test "Install alibabacloud provider" {
+    #Install csi secret driver
+    helm repo add csi-secrets-store-provider-alibabacloud https://raw.githubusercontent.com/AliyunContainerService/secrets-store-csi-driver-provider-alibaba-cloud/main/charts
+    helm install csi-secrets-store-provider-alibabacloud/csi-secrets-store-provider-alibabacloud --namespace kube-system --generate-name --set secrets-store-csi-driver.enableSecretRotation=true --set secrets-store-csi-driver.rotationPollInterval=10s  \
```

Comment:
Secrets Store CSI Driver is already installed with 

https://github.com/kubernetes-sigs/secrets-store-csi-driver/blob/350617024bf9c5f5565f7278e16c02185091aabf/Makefile#L421-L423

The provider shouldn't install the driver again as we want to validate the driver built + helm charts as part of the pull request.

Reply:
fix this